app/dashboard/views.py

```diff
@@ -1737,8 +1737,13 @@ def funder_payout_reminder(request, bounty_network, stdbounties_id):
 
 
 def quickstart(request):
-    """Display quickstart guide."""
-    return TemplateResponse(request, 'quickstart.html', {})
+    """Display Quickstart Guide."""
+
+    from dashboard.context.quickstart import quickstart
```

Comment:
Can we import this at the top of the file to make this response faster?

Reply:
Oh wouldn't that import it for every other view too / am i wrong in that assumption?

Reply:
Updated it ^_^